test/composers/moveToComposer.test.js

```diff
@@ -28,6 +28,26 @@ describe('Move To composer', () => {
 
 		chain.moveTo();
 
-		assert.deepStrictEqual(makeChain.firstCall.args[0], {isMoveTo: true});
+		assert.deepStrictEqual(makeChain.firstCall.args[0], {
+			isMoveTo: true,
+		});
+	});
+
+	it('moveTo should accept arguments and correctly translate them', () => {
```

Comment:
pls add tests that cover cases when invalid arguments passed to `moveTo`

Reply:
Done